packages/tsconfig_base.json

```diff
@@ -25,6 +25,7 @@
       "es2016",
       "es2017",
       "es2018",
+      "es2019",
```

Comment:
@michenly IIRC we chatted about doing this in the global ts configs ages ago, did we ever reach a conclusion?

Reply:
it did not, in terms of global ts config the PR was [here](https://github.com/Shopify/web/pull/24496). I pause it since Gord had brought up some really good points but I did not have time to look into.